manifests/globals.pp

```diff
@@ -227,8 +227,8 @@
         /11\.4/     => '94',
         /12\.0/     => '93',
         /12\.[1-3]/ => '94',
-        /12\.[4-5]/ => '10',
-        /15\.[0-9]/     => '10',
+        /12\.[4-5]/ => '12',
+        /15\.[0-9]/ => '14',
         default     => '96',
```

Comment:
I don't know SLES. Are point releases also supported? Is this really valid for 15.0 as well or was it only introduced in some later 15.x version?